src/main/java/com/minecolonies/util/EntityUtils.java

```diff
@@ -66,6 +70,26 @@ public static boolean isWorkerAtSiteWithMove(@NotNull EntityCitizen worker, int
      */
     public static boolean isWorkerAtSiteWithMove(@NotNull EntityCitizen worker, int x, int y, int z, int range)
     {
+        if (!isWorkerAtSite(worker, x, y, z, TELEPORT_RANGE))
+        {
+            BlockPos spawnPoint =
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'spawnPoint' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)